wallet/signer.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2020 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wallet
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcwallet/waddrmgr"
+)
+
+// Tweaker is a function type that can be used to pass in a callback for
```

Comment:
Prefix with `PrivKey`? ;) 

So `PrivKeyTweaker`, otherwise there're some undesirable connotations w.r.t the colloquial usage of this term for native english speakers lol.

Reply:
Oh, I wasn't aware of that particular colloquial usage :sweat_smile: Had to google it and now I'm kind of disappointed that "Breaking Bad" didn't teach me that term :joy:

Reply:
😂😂